tests/send_release2/ENV

```diff
@@ -0,0 +1,6 @@
+W2NM_CONTINUE_MIGRATION=false
```

Comment:
Not Sure, but maybe we should start writing minimal description of tests. So we know what we are actually looking for in this test, in 1y or so :-)

Because I was first looking for the `dhcp-send-release=1` in the `nmconnection` till I realized that this test actually covers the `<1.52` version where the option doesn't exists.